tap_instagram/streams.py

```diff
@@ -507,6 +507,10 @@ def validate_response(self, response: requests.Response) -> None:
         if (
             response.json().get("error", {}).get("error_user_title")
             == "Media posted before business account conversion"
+            or
+            str(response.json().get("error", {}).get("message"))
+            in "(#10) Not enough viewers for the media to show insights"
```

Comment:
```suggestion
            == "(#10) Not enough viewers for the media to show insights"
```
if you use `in` it will loop through each character in the string and compare, so checking if it's `== (`, `== #`, ...

Reply:
My understanding here is there is going to be additional text apart from the one I mentioned above. Here is the error I got when I faced the issue:

"400 Client Error: Bad Request - (https://github.com/prratek/tap-instagram/issues/10) Not enough viewers for the media to show insights for path: /{media_id}/insights name=meltano run_id=5f77833c-280a-4b2e-adcd-97525b0ef76b"

In this case, "Bad Request - " would be present in the error message.

Reply:
Thats why I'm a little skeptical about using "=="